src/ontology/cl-edit.owl

```diff
@@ -9456,11 +9485,12 @@ AnnotationAssertion(owl:deprecated obo:CL_0000717 "true"^^xsd:boolean)
 
 # Class: obo:CL_0000718 (compound eye cone cell)
 
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1017/CBO9780511818202.023") obo:IAO_0000115 obo:CL_0000718 "A cell of an ommatidium that secretes lens materials.")
```

Comment:
- Consider adding the exact synonym `semper cell` (and abbreviation SC).
- Consider using an additional reference that is not behind a paywall ([PMID:34343526](https://pubmed.ncbi.nlm.nih.gov/34343526)).

Reply:
No objection to adding the exact synonym, but then a comment should also be added to explain that (quite unfortunately), “Semper’s cell” may also exactly refer to a [completely different cell type in gastropods](https://journals.biologists.com/jcs/article/s3-105/71/331/63907/Semper-S-Organ-a-Cephalic-Gland-in-Certain) (also described by Carl Semper, hence the same name).